storagenode/console/service.go

```diff
@@ -29,27 +31,22 @@ var (
 
 // DB exposes methods for managing SNO dashboard related data.
 type DB interface {
-	// GetSatelliteIDs returns list of satelliteIDs that storagenode has interacted with
-	// at least once
-	GetSatelliteIDs(ctx context.Context, from, to time.Time) (storj.NodeIDList, error)
-	// GetDailyBandwidthUsed returns slice of daily bandwidth usage for provided time range,
-	// sorted in ascending order
-	GetDailyTotalBandwidthUsed(ctx context.Context, from, to time.Time) ([]BandwidthUsed, error)
-	// GetDailyBandwidthUsed returns slice of daily bandwidth usage for provided time range,
-	// sorted in ascending order for particular satellite
-	GetDailyBandwidthUsed(ctx context.Context, satelliteID storj.NodeID, from, to time.Time) ([]BandwidthUsed, error)
+	// Bandwidth is a getter for Bandwidth db
+	Bandwidth() Bandwidth
 }
 
 // Service is handling storage node operator related logic
 type Service struct {
 	log *zap.Logger
 
-	consoleDB   DB
-	bandwidthDB bandwidth.DB
-	pieceInfoDB pieces.DB
-	kademlia    *kademlia.Kademlia
-	version     *version.Service
-	nodestats   *nodestats.Service
+	trust          *trust.Pool
+	consoleDB      DB
+	bandwidthDB    bandwidth.DB
+	reputationDB   reputation.DB
+	storageusageDB storageusage.DB
```

Comment:
Would it be possible make reputationDB and storageusageDB a combination?
something like
```
type StatsCache struct {
    bandwidth  bandwidth.DB
    reputation reputation.DB
}
```

Reply:
We discussed to have separate calls with different intervals for reputation and storage usage

Reply:
They are still seperate calls, but reduce the amount of entries in the peer and personally make the structure more clear to me.
Both of them can be called and accessed independently. Its just grouping them, as they are used "together"

Reply:
Yeah, makes sense